downstream/modules/platform/ref-controller-settings-job-events.adoc

```diff
@@ -0,0 +1,9 @@
+[id="ref-controller-settings-job-events"]
+
+= Settings for managing job event processing 
+
+The callback receiver processes all the output of jobs and writes this output as job events to the {ControllerName} database. The callback receiver has a pool of workers that processes events in batches. The number of workers automatically increases with the number of CPU available on an instance.
+
+Administrators can override the number of callback receiver workers with the setting `JOB_EVENT_WORKERS`. It is not recommended to set more than 1 worker per CPU, and there must be at least 1 worker. Greater values will have more workers available to clear the Redis queue as events stream to the {ControllerName}, but can compete with other processes such as the web server for CPU seconds, uses more database connections (1 per worker), and can reduce the batch size of events each worker commits.
```

Comment:
It is not recommended to -> Do not

Greater values will have more -> Greater values have more